includes/fields/class-acf-field-nav-menu.php

```diff
@@ -0,0 +1,237 @@
+<?php
+/**
+ * This is a PHP file containing the code for the acf_field_nav_menu class.
+ *
+ * @package wordpress/secure-custom-fields
+ * @since 6.5.0
+ */
+
+if ( ! class_exists( 'Acf_Field_Nav_Menu' ) ) :
+	/**
+	 * Acf Nav menu field class
+	 */
+	class Acf_Field_Nav_Menu extends acf_field {
+
+		/**
+		 * This function will setup the field type data
+		 *
+		 * @type    function
+		 * @date    5/03/2014
+		 * @since  6.5.0
+		 */
+		public function initialize() {
+
+			// vars
+			$this->name          = 'nav_menu';
+			$this->label         = _x( 'Nav Menu', 'noun', 'secure-custom-fields' );
+			$this->category      = 'choice';
+			$this->description   = __( 'A dropdown list with a selection of menu choices that you can chose.', 'secure-custom-fields' );
+			$this->preview_image = acf_get_url() . '/assets/images/field-type-previews/field-preview-select.png';
+			$this->doc_url       = 'https://developer.wordpress.org/secure-custom-fields/features/fields/nav_menu/';
+			$this->tutorial_url  = 'https://developer.wordpress.org/secure-custom-fields/features/fields/select/nav_menu-tutorial/';
+			$this->defaults      = array(
+				'save_format' => 'id',
+				'allow_null'  => 0,
+				'container'   => 'div',
+			);
+
+			add_filter( 'acf/field_wrapper_attributes', array( $this, 'nav_menu_field_wrapper_attributes' ), 10, 2 );
+		}
+
+		/**
+		 * Renders the Nav Menu Field options seen when editing a Nav Menu Field.
+		 *
+		 * @param array $field The array representation of the current Nav Menu Field.
+		 */
+		public function render_field_settings( $field ) {
+			$allow_null = $field['allow_null'];
+			$nav_menus  = wp_get_nav_menus( $allow_null );
+			if ( current_theme_supports( 'menus' ) ) {
+				if ( empty( $nav_menus ) ) {
+					?>
+					<div class="acf-field">
+						<div class="acf-notice">
+							<p>
+								<?php esc_html_e( 'Warning: You don\'t have any menus created please visit', 'secure-custom-fields' ); ?> <a href="<?php echo esc_url( admin_url( 'nav-menus.php' ) ); ?>"><?php esc_html_e( 'this', 'secure-custom-fields' ); ?></a> <?php esc_html_e( 'link to create menus.', 'secure-custom-fields' ); ?>
```

Comment:
We'd probably want to incorporate the variable into the translation field: https://developer.wordpress.org/plugins/internationalization/how-to-internationalize-your-plugin/#variables

Additionally, can we have the actual link be more descriptive? Instead of "this", something like "This field requires an existing navigation menu, please ==create a menu first==." (`==` to show the linked text)